Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -1617,33 +1639,31 @@ namespace Babylon
     Napi::Value NativeEngine::CreateImageBitmap(const Napi::CallbackInfo& info)
     {
         const Napi::Env env{info.Env()};
+
         if (!info[0].IsArrayBuffer())
         {
             throw Napi::Error::New(env, "CreateImageBitmap parameter is not an array buffer.");
         }
 
-        const auto data = info[0].As<Napi::ArrayBuffer>();
+        const auto data{info[0].As<Napi::ArrayBuffer>()};
+
         if (!data.ByteLength())
         {
             throw Napi::Error::New(env, "CreateImageBitmap array buffer is empty.");
         }
 
-        bimg::ImageContainer* image = bimg::imageParse(&m_allocator, data.Data(), static_cast<uint32_t>(data.ByteLength()));
-        if (image == nullptr)
-        {
-            throw Napi::Error::New(env, "Unable to decode image in createImageBitmap function.");
-        }
+        bimg::ImageContainer* image{ParseImage(m_allocator, gsl::make_span(static_cast<uint8_t*>(data.Data()), data.ByteLength()))};
```

Comment:
nit: use auto everywhere suitable?

Reply:
My opinion is that we should move away from using auto unless it's completely obvious what the type is.